module/zfs/vdev.c

```diff
@@ -6031,6 +6053,7 @@ EXPORT_SYMBOL(vdev_degrade);
 EXPORT_SYMBOL(vdev_online);
 EXPORT_SYMBOL(vdev_offline);
 EXPORT_SYMBOL(vdev_clear);
+EXPORT_SYMBOL(vdev_best_ashift);
```

Comment:
Can be dropped since (most of) the Linux kmods have now been merged in to one in the master source.